client/src/modules/privacy-policy/index.jsx

```diff
@@ -0,0 +1,153 @@
+/* eslint-disable react/jsx-one-expression-per-line */
```

Comment:
why did you enable ignoring this rule here?

Reply:
with this rule we can't insert a components or html tags into the text. Eslint error occures. There is a lot of complicated texts now, some of them need special characters, spaces, sometimes we have to insert links into them. When you fix Eslit error a prettier error occures. I have no idea how to solve it in different way...